modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -60,14 +61,17 @@ spec:
               port:
                 number: 443
 ----
-
 +
 [source,terminal]
 ----
 $ oc apply -f ingress.yaml
 ----
 +
+<2> The `route.openshift.io/destination-ca-certificate-secret` can be used on an Ingress object to define a route with a custom destination certificate (CA). The annotation references a kubernetes secret, `secret-ca-cert` that will be inserted into the generated route.
```

Comment:
In my view of the preview, this `<2>` is rendering as a 3. Is that same for you? It might be that the `<2>` are used twice (Line 44 and 70)